src/game.cpp

```diff
@@ -5389,48 +5389,60 @@ void game::hallucinate( const tripoint &center )
     wrefresh(w_terrain);
 }
 
-float game::ground_natural_light_level() const
+float game::natural_light_level() const
 {
     // Already found the light level for now?
     if( calendar::turn == latest_lightlevel_turn ) {
         return latest_lightlevel;
     }
+    // Should be completely overwritten next, thus no initialization value here.
+    float ret;
```

Comment:
This assumption seems to be invalidated underground, it's generally better to initialize to something sensible no matter what's happening later.